packages/ui/v2/core/navigation/tabs/HorizontalTabItem.tsx

```diff
@@ -6,9 +6,12 @@ import { MouseEventHandler } from "react";
 import classNames from "@calcom/lib/classNames";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 
+import { PermissionContainer } from "../../PermissionContainer";
```

Comment:
HorizontalTabs should not contain outside logic. We can either create a container with logic or figure another way to achieve this